src/components/header.tsx

```diff
@@ -127,15 +127,15 @@ const Header = (props: {fixed: boolean; banner?: boolean}) => {
           <NavButton href='/#schedule'>Schedule</NavButton>
           <NavButton href='/#sponsors'>Sponsors</NavButton>
           <NavButton href='/#FAQ'>FAQ</NavButton>
-          <NavButton href='/dashboard/login'>Login</NavButton>
+          <NavButton href='/dashboard'>Dashboard</NavButton>
         </ButtonDiv>
       </NavBar>
       <NavMenu>
         <NavButton href='/#about'>About</NavButton>
         <NavButton href='/#schedule'>Schedule</NavButton>
         <NavButton href='/#sponsors'>Sponsors</NavButton>
         <NavButton href='/#FAQ'>FAQ</NavButton>
-        <NavButton href='/dashboard/login'>Login</NavButton>
+        <NavButton href='/dashboard'>Dashboard</NavButton>
```

Comment:
Would it make more sense to display "Login" if the user is not logged in?
That should be pretty easy to do with the `useAuth` hook.